pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/ForespoerselOmDokumentasjonAvBotidINorgeAlder.kt

```diff
@@ -0,0 +1,109 @@
+import no.nav.pensjon.brev.template.dsl.textExpr
+import no.nav.pensjon.brevbaker.api.model.BrukerSelectors.foedselsnummer
+import no.nav.pensjon.brevbaker.api.model.FellesSelectors.bruker
+import no.nav.pensjon.brevbaker.api.model.FellesSelectors.saksnummer
+import no.nav.pensjon.brevbaker.api.model.FoedselsnummerSelectors.value
+import no.nav.pensjon.brevbaker.api.model.LetterMetadata
+
+@TemplateModelHelpers
+object ForespoerselOmDokumentasjonAvBotidINorgeAlder : RedigerbarTemplate<EmptyRedigerbarBrevdata> {
+
+    override val kode = Brevkode.Redigerbar.PE_FORESPOERSELOMDOKUMENTASJONAVBOTIDINORGE_ALDER
+    override val kategori: TemplateDescription.Brevkategori = TemplateDescription.Brevkategori.INNHENTE_OPPLYSNINGER
+
+    override val template = createTemplate(
+        name = kode.name,
+        letterDataType = EmptyRedigerbarBrevdata::class,
+        languages = languages(Bokmal),
+        letterMetadata = LetterMetadata(
+            displayTitle = "Forespørsel om dokumentasjon av botid i Norge - alder",
+            isSensitiv = false,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VIKTIG,
+            brevtype = LetterMetadata.Brevtype.INFORMASJONSBREV,
+        )
+    ) {
+        title {
+            text(
+                Bokmal to "Forespørsel om dokumentasjon av botid i Norge"
+            )
+        }
+        outline {
+            //[PE_IY_03_167_tekst]
+            paragraph {
+                text(
+                    Bokmal to "Navn:",
+                    Element.OutlineContent.ParagraphContent.Text.FontType.BOLD
+                )
+                textExpr(
+                    Bokmal to "				".expr() + felles.bruker.fulltNavn()
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Fødselsdato:",
+                    Element.OutlineContent.ParagraphContent.Text.FontType.BOLD
+                )
+                textExpr(
+                    Bokmal to "			".expr() + felles.bruker.foedselsnummer.value
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Saksnummer:",
+                    Element.OutlineContent.ParagraphContent.Text.FontType.BOLD
+                )
+                textExpr(
+                    Bokmal to "			".expr() + felles.saksnummer,
+                )
+            }
+
+            //[PE_IY_03_167_tekst]
+
+            paragraph {
+                text(
+                    Bokmal to "Vi ber med dette om å få tilsendt oversikt over botiden (kopi av "
+                )
+                text(
+                    Bokmal to "hovedregisterkort/navnekort",
+                    Element.OutlineContent.ParagraphContent.Text.FontType.BOLD
+                )
+                text(
+                    Bokmal to " e.l) for ovennevnte person. <Fritekst: Vedkommende har opplyst at han/hun var sist bosatt i deres kommune fra <fritekst: mm.dd.år> til <<fritekst: mm.dd.år>."
```

Comment:
Før vi aktiverer dette brevet i produksjon vil jeg gjerne at vi ser på `<Fritekst ...`. Legg gjerne på en feature-toggle for brevet i skribenten.

Reply:
Lurt!